Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2098,7 +2103,7 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         //       symbols or attempt to resolve links/references since the topic graph may not contain all documents
         //       or all symbols yet.
         var result: (
-            technologies: [SemanticResult<Technology>],
+            tutorialTableOfContents: [SemanticResult<TutorialTableOfContents>],
```

Comment:
same here - make plural

Reply:
Added "Results" suffix in https://github.com/swiftlang/swift-docc/pull/1039/commits/62de8fe3c83fb7081db1ef5cdc4f7cde7ad00a7c